.jenkins/scripts/solutions/azure-config.sh

```diff
@@ -0,0 +1,5 @@
+sudo apt-get install -y azure-cli
```

Comment:
If you separate this script from the az-cli repo setup, then perhaps this line would be better in the bash script that initializes the apt repo. Then in this script you either assume azure-cli is installed, or do a check to make sure it is installed before proceeding.